src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/LuceneLocationsExtractor.scala

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.locations
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.Location
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.client.FeatureServiceClient
+import com.microsoft.partnercatalyst.fortis.spark.transforms.topic.LuceneKeyphraseExtractor
+
+class LuceneLocationsExtractor(
+                                lookup: Map[String, Set[Location]],
+                                featureServiceClient: FeatureServiceClient,
+                                placeRecognizer: Option[PlaceRecognizer] = None,
```

Comment:
Don't take this parameter if `LuceneLocationsExtractor` doesn't need it :) IMO fine to leave the plumbing in caller `LocationsExtractorFactory.create` so we can add different creation methods later, one of which may use it.